examples/onnxruntime/inceptionV3/inceptionv3.py

```diff
@@ -0,0 +1,236 @@
+        categories = [s.strip() for s in f.readlines()]
+
+    if flags.verbose:
+        print("Getting Exported Model from Torch")
+
+    # Export the model to ONNX
+    image_height = 299
+    image_width = 299
+    x = torch.randn(flags.batch, 3, image_height, image_width, requires_grad=True)
+    #torch_out = inception_v3(x)
+    torch.onnx.export(
+        inception_v3,  # model being run
+        x,  # model input (or a tuple for multiple inputs)
+        "inception_v3.onnx",  # where to save the model (can be a file or file-like object)
+        export_params=
+        True,  # store the trained parameter weights inside the model file
+        opset_version=14,  # the ONNX version to export the model to
+        do_constant_folding=
+        True,  # whether to execute constant folding for optimization
+        input_names=['input'],  # the model's input names
+        output_names=['output'])  # the model's output names
+
+    # Quantize the model
+    if flags.fp16:
+        if flags.verbose:
+            print("FP16 Quantization Enabled")
+        os.environ["ORT_MIGRAPHX_FP16_ENABLE"] = "1"  # Enable FP16 precision
+    else:
+        os.environ["ORT_MIGRAPHX_FP16_ENABLE"] = "0"  # Disable FP32 precision
+
+    session_ops = onnxruntime.SessionOptions()
+    if flags.verbose:
+        session_ops.log_verbosity_level = 0
+        session_ops.log_severity_level = 0
+
+    session_fp32 = onnxruntime.InferenceSession(
+        "inception_v3.onnx", providers=['MIGraphXExecutionProvider'], sess_options=session_ops)
+
+    if flags.verbose:
+        print("Preprocessing Batched Images")
+
+
+    # Preproccess and batch images
+    latency = []
+
+    if flags.verbose:
+        print("Read from dir " + flags.image_dir)
+        
+    fileList = get_image_list_in_dir(flags.image_dir)
+    
+    if flags.verbose:
+        print(fileList)    
+
+
+    #Setup input data feed
+    input_batch = torch.empty(flags.batch, 3,image_width,image_height)
+
+    batch_size = 0
+    for img in fileList:
+        filename = img  # change to your filename
+
+        if img == '':
+            break
+
+        if flags.verbose:
+            print("Preprocess: " + img)
+
+        input_image = Image.open(str(flags.image_dir + "/" + img))
+        preprocess = T.Compose([
+            T.Resize(342),
+            T.CenterCrop(299),
+            T.ToTensor(),
+            T.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
+        ])
+        input_tensor = preprocess(input_image)
+        input_batch[batch_size] = input_tensor.unsqueeze(0)  # create a mini-batch as expected by the model
+
+        batch_size = batch_size + 1
+        if batch_size >= flags.batch:
+            break
+
+    if flags.verbose:
+        print("Running samples")
+    output = run_sample(session_fp32, categories, latency, input_batch, flags.top, batch_size)
+
+    if flags.verbose:
+        print("Running Complete")
+        print(latency)
```

Comment:
```suggestion
       print("Total time :", latency)
```

Reply:
Done

Reply:
I don't see this change for some reason

Reply:
Batched stuff before I pushed. should be gone now

Reply:
I still don't see it